src/components/views/settings/FontScalingPanel.tsx

```diff
@@ -47,7 +47,7 @@ interface IState {
 
 export default class FontScalingPanel extends React.Component<IProps, IState> {
     private readonly MESSAGE_PREVIEW_TEXT = _t("Hey you. You're the best!");
-
+    private layoutWatcherRef!: string;
```

Comment:
This isn't right, `componentDidMount` is not guaranteed to be called.

Reply:
I have used the optional variable again and added a condition in `componentWillUnmount` to get rid of the error. Thanks